### ISSUE TITLE
Handlebars formatting will break inside styles

### ISSUE THREAD

**Issue by Razican** [beautify-web/js-beautify#1813]
# Description
This is being opened here as requested in https://github.com/microsoft/vscode/issues/101603.
It turns out that when using Handlebars inside a `<style>` HTML tag, the beautifier will break the handlebars code.

# Input
The code looked like this before beautification:
```
<style>{{ onload_style }}</style>
```

# Expected Output
The code should have looked like this after beautification:
```
<style>{{ onload_style }}</style>
```

# Actual Output
The  code actually looked like this after beautification:
```
<style>
    {
            {
            onload_style
        }
    }
</style>
```

# Steps to Reproduce

## Environment
OS: Fedora Linux using VSCode

## Settings
Default VSCode settings.

**Comment by bitwiseman** [comment]
Yes, the Css formatter does not support handlebars.  You can disable css formatting inside the style tag to mitigate this. 
But what is really needed is for the css formatter to be updated to use the same underlying parsing as the other parts of the project.  Unfortunately, I don't have time to do it currently, and it appears no one else cares enough either.
